cluster/cloud_configs/vbox.yml

```diff
@@ -1,18 +1,18 @@
 ---
 azs:
 - name: z1
+- name: z2
```

Comment:
Do we need multi-AZ for virtualbox? Seems overkill.

Reply:
Nope, we don't.

Just put it there to make the manifest look more like a "real" deployment would look like.

Ill revert this.